example/virtual_functions/Makefile

```diff
@@ -1,5 +1,5 @@
 KOKKOS_DEVICES=Cuda
-KOKKOS_CUDA_OPTIONS=enable_lambda
+KOKKOS_CUDA_OPTIONS=enable_lambda,rdc
```

Comment:
Does this belong in this pull request?

Reply:
i leave that in here since this change wouldn't get tested in PR testing anyway, and I don't want to waste the cycles

Reply:
but without it the code doesn't actually compile ....

Reply:
> i leave that in here since this change wouldn't get tested in PR testing anyway, and I don't want to waste the cycles

I don't quite agree with that philosophy personally but won't insist.